examples/grid/rl_grid_direct_mpc_curr_ctr.py

```diff
@@ -24,20 +24,21 @@
 from soft4pes.sim import Simulation
 
 # Define base values
-base = model.grid.BaseGrid(Vgr=3300, Igr=1575, fgr=50)
+base = model.grid.BaseGrid(Vg_R_SI=3300, Ig_R_SI=1575, fg_R_SI=50)
 
 # Define current reference sequence
 i_ref_dq = Sequence(np.array([0, 0.1, 0.1, 1]),
                     np.array([[1, 0], [1, 0], [0.3, 0.3], [0.3, 0.3]]))
 
+# Define grid parameters
+grid_params = model.grid.RLGridParameters(Vgr_SI=3300,
```

Comment:
What does the subscript ``r`` stand for?

Reply:
This has been removed in the next commit, the grid values can deviate from the rated ones